src/test/java/com/github/fge/jsonpatch/diff/JsonDiffTest.java

```diff
@@ -87,21 +87,33 @@ public void generatedPatchAppliesCleanly(final JsonNode first,
             list.add(new Object[] {
                 node.get("message").textValue(), node.get("first"),
                 node.get("second"), node.get("patch"),
-                node.has("diffDoesntRequireSource") ? node.get("diffDoesntRequireSource").booleanValue() : DIFF_DOESNT_REQUIRE_SOURCE
+                node.has("options") ? getLiteralOptions(node.get("options")) : DiffOptions.DEFAULT_OPTIONS
             });
         }
 
         return list.iterator();
     }
 
+    public DiffOptions getLiteralOptions(JsonNode jsonNode) {
+        DiffOptions.Builder builder = new DiffOptions.Builder();
+        if (jsonNode.has("diffDoesntRequireSource")) {
```

Comment:
Maybe try to avoid magic strings ?

Reply:
it's just for access to json tests values...